### DISCUSSION TITLE
New Reticulum Identity

### DISCUSSION THREAD

**Discussion by Heavyclouds** [markqvist/Reticulum#366] [Help & Questions] [answered]
I have created two rnodes and am using them from cloned bootable drives. I noticed I have the same reticulum ids for both. Where can I change or how do I generate a new id for the host?

**Comment by faragher** [comment, marked as answer]
This presumes you cloned the drives as a starting point and are happy to modify the file systems independently.

Identities are application specific. If you want to create a new identity for NomadNet for instance, you can move or delete ~/.nomadnetwork/storage/identity

There's an identity for RNSH under ~/.reticulum/storage/identities/rnsh.default

If you're writing your own application, instead of loading your identity, you can simply create a new one with RNS.Identity()

What application(s) did you want to have a new identity?

**Reply by Heavyclouds** [reply]
Correct. I cloned so I didn’t have to go through the trouble of creating a new bootable with all the utilities. I was trying to send messages through Sideband and Nomadnet. The Nomadnet host could see both messages, but the identities were the same. 

I’m trying to mirror the instructions on Unsigned.io for messaging as a starting point, which is when I noticed the identity in Nomadnet on pc1 and Sideband on pc2 were the same. 

If I delete the Nomadnet identity and/or the rnsh identity, when I reboot or restart Nomadnet will it create a new one? Or is there additional steps after?

**Reply by faragher** [reply]
I'm not 100% certain, but since it generates an identity on first run, I'd bet on it generating a new identity if one is missing.

**Reply by Heavyclouds** [reply]
I’ll give it a shot. Thank you! Do you know where I can do this for Sideband?

**Reply by Heavyclouds** [reply]
It worked for Nomadnet and I did the same for Sideband at ~/.config/sideband/app_storage

**Comment by markqvist** [comment]
You can also use the `rnid` utility to easily generate new identities. For example, to generate a new identity for nomadnet (overwriting the existing one):

```bash
rnid --generate ~/.nomadnetwork/storage/identity --force
```
